packages/plugin-ext/src/plugin/plugin-context.ts

```diff
@@ -182,7 +182,16 @@ export function createAPIFactory(
     return function (plugin: InternalPlugin): typeof theia {
         const commands: typeof theia.commands = {
             // eslint-disable-next-line @typescript-eslint/no-explicit-any
-            registerCommand(command: theia.CommandDescription, handler?: <T>(...args: any[]) => T | Thenable<T | undefined>, thisArg?: any): Disposable {
+            registerCommand(command: theia.CommandDescription | string, handler?: <T>(...args: any[]) => T | Thenable<T | undefined>, thisArg?: any): Disposable {
+                // use of the ID when registering commands
+                if (typeof command === 'string') {
+                    const rawCommands = plugin.rawModel.contributes && plugin.rawModel.contributes.commands;
+                    const commands = rawCommands ? Array.isArray(rawCommands) ? rawCommands : [rawCommands] : undefined;
```

Comment:
@JonasHelming please use a different variable name, the following has a CI [error](https://travis-ci.com/eclipse-theia/theia/jobs/295798297#L827).

Reply:
done